src/PeopleLookup.Mvc/Controllers/IamwsController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Ietws;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace PeopleLookup.Mvc.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class IamwsController : ControllerBase
+    {
+        // GET: api/Iamws
+        [HttpGet]
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET: http://localhost:53259/api/Iamws/PPSAssociation/xxxx/iamId/default/1000012183
+        [HttpGet("PPSAssociation/{key}/{search}/{retType}/{id}", Name = "GetPPSAssociation")]

```

Comment:
@jSylvestre I think it'd be better to keep most of these as query params, not route params.  it'll keep it more aligned with the IETWS but also it's much more flexible and allows for defaults.  So I'd change the httpGet to just `[HttpGet("PPSAssociation", Name = "GetPPSAssociation")]`.  Next, the final param can't be Id since it's not always an id, it depends on the search field.  So I think it'd just change the search param to `searchField` and then your string id param to string `searchValue`.  That mirrors what the SDK is using. And then optionally you can set the retType default if you want, up to you on that one.